assets/NonFunctionalRequirements

```diff
@@ -0,0 +1,8 @@
+Kwaliteitseisen
+
+Non-functionele eisen:
+1. Gebruikersgegevens moeten beveiligd zijn
+2. Het systeem moet bruikbaar zijn voor alle medewekers/admins tegelijk
+3. De servers moeten goed functioneren bij zowel een groot als klein aantal gebruikers
+
+Deze vond ik wat lastiger, graag wat meer voorbeelden in de feedback.
```

Comment:
Niet-functionele eisen zijn alle eisen die niet te maken hebben met de functionaliteit van de applicatie. 
Als dat wat vaag klinkt kun je ook kijken dat de niet-functionele eisen in een van de volgende categorien past: 

    Prestaties
    Beveiliging
    Betrouwbaarheid
    Bruikbaarheid
    Schaalbaarheid
    Compatibiliteit

De eisen die jij gemaakt hebt vallen onder die categorien, maar ze voldoen nog niet allemaal aan de INVEST regels.